railties/lib/rails/generators/rails/plugin/templates/Gemfile.tt

```diff
@@ -1,36 +1,19 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+<% unless options[:skip_gemspec] -%>
 
-<% if options[:skip_gemspec] -%>
-<%= "# " if rails_prerelease? -%>gem "rails", "<%= Array(rails_version_specifier).join("', '") %>"
-<% else -%>
 # Specify your gem's dependencies in <%= name %>.gemspec.
 gemspec
 <% end -%>
-<% unless options[:skip_active_record] -%>
-
-group :development do
```

Comment:
Will the database gem now be in the default group?

Reply:
Yes, but it is not added as a runtime dependency because it is not in the `*.gemspec` file.  It is the same as the [asset pipeline gemfile entry](https://github.com/rails/rails/commit/f292daad7051275f7b557b352876cfeb3988f639#diff-479423715f9f660607c18ea3d6bfd68b8f7b8fbaf7148937a0fd4dc76627d9f2R18) from f292daad7051275f7b557b352876cfeb3988f639.